vars/mtaBuild.groovy

```diff
@@ -35,7 +35,9 @@ import static com.sap.piper.Utils.downloadSettingsFromUrl
 ]
 @Field Set PARAMETER_KEYS = STEP_CONFIG_KEYS.plus([
     /** @see dockerExecute */
-    'dockerOptions'
+    'dockerOptions',
+    /** Url to the npm registry that should be used for installing npm dependencies.*/
+    'npmRegistry'
```

Comment:
We could name it defaultNpmRegistry as in https://github.com/SAP/jenkins-library/blob/master/vars/npmExecute.groovy#L17.

The idea behind it is that you still can overwrite it or have additional custom registries.

Reply:
done